### ISSUE TITLE
Insider reflow validation: Expanded/Collapsed states of the tests in left nav are not announced by screen readers

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#3100]
## Describe the bug

When navigating through the left nav in assessment with screen readers, expand/collapse state of the "test" list is not announced by screen readers

Browser | AT
-- | --
Chrome | NVDA
Edge | Narrator
Edge | JAWS

**To Reproduce**

1. Run assessment on 'https://www.washington.edu/accesscomputing/AU/before.html'
2. Use one of the above browser/AT combination to navigate to the left nav
3. Press enter on the tests to expand and collapse
4. Observe that the expanded/collapsed state of the tests are not announced by screen readers

## Expected behavior

After expanding or collapsing a test menu, the state should be announced by the screen readers

## Screenshots

![2](https://user-images.githubusercontent.com/22068434/87811385-7751ba00-c813-11ea-9d24-7a7cfcc6404f.gif)

## Context
- OS Name: Windows 10, Version: 1903, Build 18362.476
- AI-Web Version & Environment: Insider 2.19.0 
- Browser Version: Google Chrome 84.0.4147.89, Microsoft Edge 83.0.478.64
- Target Page: https://www.washington.edu/accesscomputing/AU/before.html

**Comment by ferBonnin** [comment]
@waabid assigning to you and marking this as active

**Comment by iamrafan** [comment]
verified in v2.19.3
